lib/palindrome_permutation.rb

```diff
@@ -1,5 +1,20 @@
 
 def palindrome_permutation?(string)
-  raise NotImplementedError, "palindrome_permutation? not implemented"
-end
+  hash = {}
+  array = string.split ("")
```

Comment:
Just a note, Ruby has a `.each_char` method, so you don't **have** to split the string.